tests/test_environment.py

```diff
@@ -15,20 +16,28 @@
 # to do it.
 
 
-def test_mail(mailoutbox):
-    assert mailoutbox is mail.outbox  # check that mail.outbox and fixture value is same object
-    assert len(mailoutbox) == 0
+def test_direct_mailbox_access_not_allowed():
+    with pytest.raises(AssertionError):
+        len(mail.outbox)
+
+    with pytest.raises(AssertionError):
+        mail.outbox[0]
+
+    with pytest.raises(AssertionError):
+        if mail.outbox:
+            pass
+
+
+def test_direct_mailbox_proection_should_not_break_sending_mail():
     mail.send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
-    assert len(mailoutbox) == 1
-    m = mailoutbox[0]
-    assert m.subject == 'subject'
-    assert m.body == 'body'
-    assert m.from_email == 'from@example.com'
-    assert list(m.to) == ['to@example.com']
 
 
-def test_mail_again(mailoutbox):
-    test_mail(mailoutbox)
+class TestDirectAccessWorksForDjangoTestCase(TestCase):
+
+    def test_one(self):
+        assert len(mail.outbox) == 0
+        mail.send_mail('subject', 'body', 'from@example.com', ['to@example.com'])
+        assert len(mail.outbox) == 1
```

Comment:
Please add test_two that does the same thing and checks to make sure the clearing works :)